staging/src/github.com/kubeedge/mapper-framework/pkg/util/parse/grpc.go

```diff
@@ -133,64 +133,66 @@ func buildPropertiesFromGrpc(device *dmiapi.Device) []common.DeviceProperty {
 			dbMethodName, err = getDBMethodFromGrpc(pptv)
```

Comment:
It doesn't look like the function `getDBMethodFromGrpc(..)` is playing a big role here, we can do this directly with the switch:
```golang
switch pptv
case visitor.PushMethod.DBMethod.Influxdb2 != nil:
    ...
case visitor.PushMethod.DBMethod.Redis != nil:
    ...
case visitor.PushMethod.DBMethod.Tdengine != nil:
    ...
default:
    klog.Error("...")

Reply:
Done